dbfit-java/build.gradle

```diff
@@ -83,7 +83,80 @@ subprojects {
         include('**/*Support*')
     }
 
-    task integrationTest(type: Test) {
+    task deploymentProperties {
+        doLast {
+            ext {
+                jdbcDriverClass = ''
+                jdbcDriverUrlPrefix = ''
+                delimeterType = 'normal'
+                delimeter = ';'
+                setUpTestDbScript = "${rootProject.projectDir}/test_vm/scripts/${project.name}/sql/create-databases-${project.name}.sql"
+                tearDownTestDbScript = "${rootProject.projectDir}/test_vm/scripts/${project.name}/sql/drop-databases-${project.name}.sql"
+            }
+        }
+    }
+
+    // to be overriden when needed
+    ext.connectionPropertiesFile = '/dev/null'
```

Comment:
Is it needed and safe to have default file here? What would happen if it's not overriden? (I'm not sure whether `/dev/null` is available on Windows).